scorpius_manipulation_demo/SceneObject[] objects